metadata/src/test/java/org/apache/kafka/controller/errors/EventHandlerExceptionInfoTest.java

```diff
@@ -72,6 +73,11 @@ public class EventHandlerExceptionInfoTest {
             new TimeoutException(),
             () -> OptionalInt.of(1));
 
+    private static final EventHandlerExceptionInfo KAFKA_EXCEPTION =
+        EventHandlerExceptionInfo.fromInternal(
+            new KafkaException("Custom kafka exception message"),
+            () -> OptionalInt.of(1));
+
```

Comment:
It seems this is only used in `testKafkaExceptionFailureMessage`, can we move it there?

Reply:
Initially I thought it would be better to keep all `EventHandlerExceptionInfo` declarations at the same place. But as you mentioned, you would be even better to have it on a method level as it is used only once. Moved `TIMEOUT` to the method too